certified-connectors/DeskDirector Tech/README.md

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Could you please update readme.md file as per the template in the below link?

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md